service/static/js/index.js

```diff
@@ -180,7 +180,7 @@ $(function () {
 
     ajax.done(function(res){
       // TODO: remove this ternary after READ is moved to restx too
-      build_wishlist_results_table(wishlist_id ? res['data'] : res);
+      build_wishlist_results_table(wishlist_id ? res : res);
```

Comment:
No need to check anymore, so this should just be `build_wishlist_results_table(res);`